### PR TITLE
feat(ui): beta badge

### PR DESCRIPTION
**Pull request by ckj**
Replaces the `previewFeature` alert with a fancy new Beta badge + tooltip.

screen shot 2018-11-19 at 5 02 16 pm

cc @benvinegar

### CODE REVIEW

**Comment by billyvg** [comment]
We should probably add this to Discover as well

**Comment by EvanPurkhiser** [comment]
😍

**Comment by ckj** [comment]
> We should probably add this to Discover as well

Agreed, waiting on #10671

**Comment by ckj** [comment]
this is now ready for review

**Review by Chrissy — APPROVED** [review]
Looks good. I added some commentz but they are all code style suggestions.

File: src/sentry/static/sentry/app/components/betaTag.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    </span>
+  </Tooltip>
+);
+
+const StyledTag = styled(Tag)`
+  position: relative;
+  top: -1px;
+  font-size: 12px;
+  font-weight: normal;
+  padding: 3px 6px;
```
[the remark this answers is not in the document]
Comment (ckj) [reply src/sentry/static/sentry/app/components/betaTag.jsx]: yeah... 2/4px not enough, 4/8px too much

File: src/sentry/static/sentry/app/components/betaTag.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  margin-left: ${space(0.5)};
+  border-radius: 20px;
```
[the remark this answers is not in the document]
Comment (ckj) [reply src/sentry/static/sentry/app/components/betaTag.jsx]: Nope, just a high enough number to get a perfect pill shape. I usually do border-radius: 50%, but in this case it's giving me a weird squashed oval effect

**Pull request merged**